src/App.css

```diff
@@ -1,39 +1,48 @@
-.App {
-  text-align: center;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+body {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
```

Comment:
- [ ] I understand that working on the design may be frustrating and I congratulate you to have a very nice design. However, as the previous reviewers suggest, kindly try as much as possible to stick to the provided calculator design. You should atleast make sure that the zero is aligned to the center right of the calculator to match the original design.
- [ ] In addition, kindly make sure that the width is relatively larger than the height. This will help us to make a professionally designed calculator.
![image](https://user-images.githubusercontent.com/99394374/234542720-f7c9009a-6d04-410d-aaf3-aa8b967fbc4e.png)